ProcessMaker/modeler
# FOUR-8393: Elements are moved out of the Pool container

## Issue & Reproduction Steps

Expected behavior: 
The elements should not be moved out of the Pool container

Actual behavior: 
The elements copied are moved  out of the Pool container
## Solution
- saveState logic was moved to the selection and drag side
- flows into container will be updated after the stopDrag

https://github.com/ProcessMaker/modeler/assets/1401911/518f1eee-830a-4383-8cd1-aa320660d82b

https://github.com/ProcessMaker/modeler/assets/1401911/95964ec0-f610-4ace-8343-dd46e69bc6f8


## How to Test
Test the steps above
- npm install
- npm run serve
- check the steps in the related tickets

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-8393
- https://processmaker.atlassian.net/browse/FOUR-8651

## Code Review Checklist
